src/components/follow/Follow.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { Story, Meta } from '@storybook/react';
+import Follow from './Follow';
+import Props from './Follow.types';
+
+export default {
+  title: 'Components/Follow',
+  component: Follow,
+  argTypes: {
+    status: {
+      options: ['DRAFT', 'PENDING', 'IN_REVIEW', 'REVIEWED', 'REJECTED', 'APPROVED', 'COMPLETED'],
```

Comment:
Here to make this dynamic you can import Project status from status.ts and do something like this:
Object.values(PROJECT)

Reply:
fixed